src/core/book-keeper.js

```diff
@@ -1,9 +1,277 @@
 'use strict';
 
-class BookKeeper {
-    constructor () {
+var Util = require('./util');
 
+var moment = require('moment');
+
+var metricsEventEmitter = require('../src/core/event/event-bus.js').EventEmitter;
+
+
+var init = function(EventI) {
```

Comment:
Let's stick with the convention set by the rest of the project and name the event bus "eventBus" instead of "EventI"